.changeset/forty-planes-approve.md

```diff
@@ -0,0 +1,120 @@
+"@twilio-paste/aspect-ratio": major
+"@twilio-paste/flex": major
+"@twilio-paste/grid": major
+"@twilio-paste/media-object": major
+"@twilio-paste/stack": major
+"@twilio-paste/box": major
+"@twilio-paste/combobox-primitive": major
+"@twilio-paste/disclosure-primitive": major
+"@twilio-paste/listbox-primitive": major
+"@twilio-paste/menu-primitive": major
+"@twilio-paste/modal-dialog-primitive": major
+"@twilio-paste/non-modal-dialog-primitive": major
+"@twilio-paste/sibling-box": major
+"@twilio-paste/tabs-primitive": major
+"@twilio-paste/text": major
+"@twilio-paste/tooltip-primitive": major
+"@twilio-paste/customization": major
+"@twilio-paste/icons": major
+"@twilio-paste/animation-library": major
+"@twilio-paste/clipboard-copy-library": major
+"@twilio-paste/code-editor-library": major
+"@twilio-paste/data-visualization-library": major
+"@twilio-paste/dropdown-library": major
+"@twilio-paste/lexical-library": major
+"@twilio-paste/react-spectrum-library": major
+"@twilio-paste/react-textarea-autosize-library": major
+"@twilio-paste/reakit-library": major
+"@twilio-paste/styling-library": major
+"@twilio-paste/syntax-highlighter-library": major
+"@twilio-paste/uid-library": major
+"@twilio-paste/style-props": major
+"@twilio-paste/theme": major
+"@twilio-paste/types": major
+"@twilio-paste/utils": major
+"@twilio-paste/token-contrast-checker": minor
+---
+
+[account-switcher, ai-chat-log, alert, alert-dialog, anchor, avatar, badge, base-radio-checkbox, blockquote, breadcrumb, button, button-group, callout, card, chart-provider, chat-composer, chat-log, checkbox, code-block, combobox, corner-ornament, data-grid, date-picker, description-list, detail-text, disclosure, display-heading, display-pill-group, editable-code-block, example-text, file-picker, file-uploader, form, form-pill-group, heading, help-text, in-page-navigation, inline-code, inline-control-group, input, input-box, keyboard-key, label, list, menu, meter, minimizable-dialog, modal, page-header, pagination, paragraph, popover, product-switcher, progress-bar, progress-steps, radio-button-group, radio-group, screen-reader-only, select, separator, side-modal, side-panel, sidebar, skeleton-loader, slider, spinner, status, summary-detail, switch, table, tabs, textarea, time-picker, timeline, toast, tooltip, topbar, truncate, user-dialog, visual-picker, core, aspect-ratio, flex, grid, media-object, stack, box, combobox-primitive, disclosure-primitive, listbox-primitive, menu-primitive, modal-dialog-primitive, non-modal-dialog-primitive, sibling-box, tabs-primitive, text, tooltip-primitive, customization, icons, animation-library, clipboard-copy-library, code-editor-library, data-visualization-library, dropdown-library, lexical-library, react-spectrum-library, react-textarea-autosize-library, reakit-library, styling-library, syntax-highlighter-library, uid-library, style-props, theme, token-contrast-checker, types, utils]: Upgrade to react 19 and drop support for react 16
```

Comment:
```suggestion
[all packages]: Upgrade to React 19 and drop support for React 16. Paste only supports the three latest versions of React. If you're using Paste in a React 17, 18, or 19 app, you should be able to upgrade with full backwards compatibility. If you experience any issues, please let us know on Github.
```

Reply:
@nkrantz yeah so token-contrast-checker never had a major release yet, its version is 0.x.x so thought just upgrading the react version is proabably not a valid reason for major release. Should I move it to major release?